### PR TITLE
Add initial gentx instructions

### PR DESCRIPTION
**Pull request by dylanschultzie**
Signed-off-by: Dylan Schultz <hello@schultzie.dev>

### CODE REVIEW

**Comment by dylanschultzie** [comment]
@nullmames

**Review by nullmames — CHANGES_REQUESTED** [review]
see inline comments

File: defund-private-2/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+```sh
+git clone https://github.com/defund-labs/defund
+cd defund
+git checkout v0.1.0
+make install
+```
+
+#### 2. Init chain
+```sh
+defundd init $MONIKER_NAME --chain-id defund-testnet-2
```
[the remark this answers is not in the document]
Comment (dylanschultzie) [reply defund-private-2/README.md:76]: @schnetzlerjoe can you confirm please?

File: defund-private-2/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+```sh
+git clone https://github.com/defund-labs/defund
+cd defund
+git checkout v0.1.0
+make install
+```
+
+#### 2. Init chain
+```sh
+defundd init $MONIKER_NAME --chain-id defund-private-2
```
Comment (nullmames) [inline defund-private-2/README.md:76]: $MONIKER_NAME variable has not yet been set. change to `<moniker>` and add note to replace.

File: defund-private-2/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (schnetzlerjoe) [reply defund-private-2/README.md]: defund-private-2 is correct

**Review by schnetzlerjoe — COMMENTED** [review]
Yeah just need to change the moniker and not sure if we want to change 
```bash
git checkout v0.1.0
```
to
```bash
git checkout v0.1.0-alpha
```
for now until the official release of v0.1.0 prob in a few days or next week?

**Comment by nullmames** [comment]
Thanks @dylanschultzie

**Pull request merged**